hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/StochasticLoadBalancer.java

```diff
@@ -1105,6 +1114,28 @@ int regionIndexToEntityIndex(int region) {
     }
   }
 
+  static class ServerSsdLocalityCostFunction extends LocalityBasedCostFunction {
+
+    private static final String LOCALITY_COST_KEY = "hbase.master.balancer.stochastic.ssdLocalityCost";
+    private static final float DEFAULT_LOCALITY_COST = 25;
+
+    ServerSsdLocalityCostFunction(Configuration conf, MasterServices srv) {
+      super(
+        conf,
+        srv,
+        LocalityType.SERVER,
+        true,
+        LOCALITY_COST_KEY,
+        DEFAULT_LOCALITY_COST
+      );
```

Comment:
Please use style of surrounding code. Params are on one line, not a line per param.

Reply:
fix